pkg/resource/events-logger-config/alloy/events-logger.alloy.template

```diff
@@ -80,19 +80,41 @@ otelcol.processor.k8sattributes "default" {
       "k8s.namespace.name",
       "k8s.pod.name",
       "k8s.container.name",
+	  "k8s.pod.labels"
     ]
     otel_annotations = true
   }
 
   output {
-    traces = [otelcol.exporter.otlphttp.default.input]
+    traces = [
+		{{- range .Tenants }}
+		otelcol.processor.filter.{{ . }},
```

Comment:
This outputs wrong:

output {
    traces = [ 
    otelcol.processor.filter.giantswarm,
  ]